core/modules/qdisp/QueryRequest.cc

```diff
@@ -153,6 +154,13 @@ bool QueryRequest::_importStream() {
     bool retrieveInitiated = false;
     // Pass ResponseRequester's buffer directly.
     std::vector<char>& buffer = _requester->nextBuffer();
+    LOGF_DEBUG("QueryRequest::_importStream buffer.size=%1%" % buffer.size());
+    for(auto iter=buffer.begin(); iter != buffer.end(); iter++) *iter=8;
```

Comment:
Deleted the line. Just wanted a non-zero value so we could see if buffer was being altered.